pkgs/test_core/lib/src/runner/configuration/args.dart

```diff
@@ -82,9 +82,12 @@ final ArgParser _parser = (() {
       defaultsTo: '30s');
   parser.addFlag("pause-after-load",
       help: 'Pauses for debugging before any tests execute.\n'
-          'Implies --concurrency=1 and --timeout=none.\n'
+          'Implies --concurrency=1, --debug, and --timeout=none.\n'
           'Currently only supported for browser tests.',
       negatable: false);
+  parser.addFlag("debug",
+      help: 'Runs browsers in debug mode.',
```

Comment:
Is this accurate for browsers that aren't Chrome?

Reply:
Updated